docs/DOCS.md

```diff
@@ -0,0 +1,835 @@
+| `pcds_dsn` | PCDS database URL of the form `dialect[+driver]://username:password@host:port/database`. Password must either be supplied or available in the user's `~/.pgpass` file. |
+| `js_min` | Determine's use of javascript bundling/minification. |
+| `geoserver_url` | PCDS Geoserver URL |
+| `ncwms_url` | Raster portal ncWMS URL |
+| `tilecache_url` | Tileserver URLs (space separated list) for base maps |
+| `use_analytics` | Enable or disable Google Analytics reporting |
+| `analytics` | Google Analytics ID |
+
+## 3.2 JavaScript Configuration Code
+
+Some portals are configured by hard-coded values in the client
+app JavaScript.
+Other portals are configured a separate JS configuration file that exports
+a configuration object processed by the client app.
+
+A separate configuration file can easily be superseded by mounting a volume
+to its file path that contains different configuration content. In the Docker
+container, such files have internal (target) file paths of the form
+`/root/pdp/pdp/static/js/<portal>_config.js`; for example,
+`/root/pdp/pdp/static/js/prism_demo_config.js`.
+Note the doubled `pdp` subdirectories.
+
+Developers are **strongly encouraged** to keep the JS configuration files in
+this repo up to date with the most recently deployed configurations. When
+a configuration is changed for deployment, the repo copy of the configuration
+file should also be changed appropriately. A new release need not be made
+right away (that is of course the point of separate configuration),
+but eventually updates will make their way into releases, and we will also
+have a typical or standard configurations that are easily accessible.
+
+At present, the following JS portal configuration files exist:
+- PRISM: `pdp/static/js/prism_demo_config.js`
+
+# 4.0 Tests
+
+## 4.1 Python (Docker local pytest environment)
+
+### 4.1.1 TL;DR: Brief instructions
+
+For greater detail, see section Instructions below.
+
+1. Pull the image 
+
+   `docker pull pcic/pdp-local-pytest:<tag>`
+   
+   where `<tag>` either `latest` or your branch name, the latter only if
+   you modified the Dockerfile in `local-pytest`. You may only rarely need
+   to pull a new copy of the image.
+   
+1. Run the container: `./docker/local-pytest/up-backend.sh`. The container will
+   start, install your local version of the project, and give you a bash
+   command line prompt.
+    
+1. Run your tests, etc.: In the running container, enter commands at the prompt,
+   e.g., `py.test ...`.
+   
+1. Stop the container: When you no longer wish the container to be running,
+   enter `ctrl+D` or `exit` at the container command line prompt. 
+   
+   If you detached from the image, you can enter 
+   `./docker/local-pytest/down-backend.sh` to stop and remove the container.
+
+### 4.1.2 What
+
+The files in this directory allow you to build and run a test environment
```

Comment:
This README is no longer in the same directory as the tests. May need to specify which directory here.